### PR TITLE
doc: matter: improvements to Adding clusters page

### PR DESCRIPTION
**Pull request by markaj-nordic**
Added minor improvements to clarify ZAP tool dependencies.

Signed-off-by: Marcin Kajor <marcin.kajor@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/ug_matter_gs_adding_clusters.rst [hunk 1]
```diff
@@ -94,14 +94,18 @@ This tool is provided with the Matter submodule.
...
+#. Open your installation directory for the |NCS| in a command line.
 #. Navigate to :file:`modules/lib/matter`.
 #. Open the :file:`src/template.zap` for editing by running the following command, where ``samples/matter/sensor`` stands for the path where you copied the template sample in the first step of this guide.
 
    .. code-block::
 
       ./scripts/tools/zap/run_zaptool.sh ../../../nrf/samples/matter/sensor/src/template.zap
 
+  .. note::
+      The :file:`run_zaptool.sh` script will run the bootstrap procedure to install all required ZAP tool dependencies (nodejs packages).
```
Comment (greg-fer) [inline doc/nrf/ug_matter_gs_adding_clusters.rst:107]: What dependencies do you mean here? Isn't their installation already covered by step 1 of this section?

Comment (markaj-nordic) [reply doc/nrf/ug_matter_gs_adding_clusters.rst:107]: `run_zaptool.sh` will only install nodejs packages, nodejs itself must be preinstalled.

File: doc/nrf/ug_matter_gs_adding_clusters.rst [hunk 2]
```diff
@@ -94,14 +94,18 @@ This tool is provided with the Matter submodule.
 
 To edit clusters using the ZAP tool, complete the following steps:
 
-1. Open your installation directory for the |NCS| in a command line.
+1. Make sure that you have installed the ZAP tool dependencies, as mentioned in the Quick instructions section in the :file:`README.md` file in the official `ZCL Advanced Platform`_ repository.
```
Comment (greg-fer) [inline doc/nrf/ug_matter_gs_adding_clusters.rst:97]:
```suggestion
1. Make sure that you have installed the ZAP tool prerequisites, as mentioned in the Quick instructions section in the :file:`README.md` file in the official `ZCL Advanced Platform`_ repository.
```

**Review by greg-fer — APPROVED** [review]
Approving, but please apply the comment.

File: doc/nrf/ug_matter_gs_adding_clusters.rst (hunk 2 above)
Comment (markaj-nordic) [reply doc/nrf/ug_matter_gs_adding_clusters.rst]: Thanks, done.

File: doc/nrf/ug_matter_gs_adding_clusters.rst [hunk 3]
```diff
@@ -94,14 +94,18 @@ This tool is provided with the Matter submodule.
...
+#. Open your installation directory for the |NCS| in a command line.
 #. Navigate to :file:`modules/lib/matter`.
 #. Open the :file:`src/template.zap` for editing by running the following command, where ``samples/matter/sensor`` stands for the path where you copied the template sample in the first step of this guide.
 
    .. code-block::
 
       ./scripts/tools/zap/run_zaptool.sh ../../../nrf/samples/matter/sensor/src/template.zap
 
+  .. note::
+      The :file:`run_zaptool.sh` script will run the bootstrap procedure to install all required ZAP tool dependencies (Node.js packages).
```
Comment (greg-fer) [inline doc/nrf/ug_matter_gs_adding_clusters.rst:107]: This block is misaligned. Add one space to fix it:
```suggestion
   .. note::
       The :file:`run_zaptool.sh` script will run the bootstrap procedure to install all required ZAP tool dependencies (Node.js packages).
```

**Review by carlescufi — APPROVED** [review]

**Pull request merged**